e2e/etcdscale_test.go

```diff
@@ -132,8 +132,8 @@ func resizeSelfHostedEtcd(c kubernetes.Interface, size int) error {
 		return nil
 	}
 
-	if err := retry(15, 10*time.Second, podsReady); err != nil {
-		return fmt.Errorf("Waited 150 seconds for etcd to scale: %v", err)
+	if err := retry(31, 10*time.Second, podsReady); err != nil {
+		return fmt.Errorf("Waited 300 seconds for etcd to scale: %v", err)
```

Comment:
You would think, but the retry function actually will do N retries and N-1 pauses. So `try -> pause -> try -> pause -> try` It doesn't pause after the last try.